### ISSUE TITLE
Perceptron and svcca sample space considerations

### ISSUE THREAD

**Issue by danielmk** [barisckuru/phase-to-rate#22]
As discussed in #9 the large scale simulations are currently running and as @barisckuru pointed out we should consider the amount of samples we need for both the perceptron validation and svcca.

The current simulations generate 2000 poisson seeds for 5 trajectories making for a total of 10000 samples and 200000 bins (10000*20).

**Regarding svcca:**
For svcca I think we will have enough samples. The svcca docs (https://github.com/google/svcca/blob/master/tutorials/001_Introduction.ipynb) read:
```
It's not always feasible to use enormous numbers of datapoints (for example, if there are memory constraints.) But two useful rules of thumb are:

Pick the number of datapoints to be at least a few times larger than the number of neurons.
Compute a baseline with your number of datapoints, where CCA correlation should be low, and analyze results relative to this.
```

We have 2000 granule cells, 200 grid cells and two coordinates for each cell making for a total of 4400 "neurons". 10000 samples is already more than 2 times larger, which is a good start. Moreover, I had the thought that we present the same set of neurons 20 different "stimuli" (locations) during each sample, hence 200000 bins and I think that should be our sample space. That would be plenty enough samples.

**Regardin perceptron validation**:
The perceptron is more problematic. One reason is that we have at maximum 2000 * 20 * 2 (neurons, bins, coordinates) input features but we also have 2 output neurons making for 2000 * 20 * 2 * 2 (160000) weights. So we definitely can only train on single bins, which brings us back to 2000 * 2 * 2 (8000).

I don't think for the perceptron we can use the same logic as for svcca and use multiple bins because that changes the learning task from identifying trajectories to identifying location but I did not yet think through it completely. I am not sure yet how to handle the perceptron validation.

I am not even sure yet if we can classify all 10000 samples with the same trajectory without increasing the output layer size and thereby again increasing the number of weights.

In summary I think we will be fine regarding svcca but the perceptron validation really requires a massive number of samples I am not yet sure how to generate.

Any thoughts @barisckuru ?
